solver/src/driver.rs

```diff
@@ -350,10 +350,13 @@ impl Driver {
         }
 
         let rated_settlements = self.rate_settlements(settlements, &estimated_prices).await;
+        // TODO: 1e18 should be 10**(native_token_decimals)
+        let gas_price_normalized: BigRational = estimated_prices.get(&self.native_token).unwrap()
+            / BigRational::from_integer(1_000_000_000_000_000_000_u128.into());
```

Comment:
Looking at the implementation of `collect_estimated_prices` I don't think it is guaranteed to contain `native_token`. We do however calculate prices relative to the native token 
```rust
let denominator_token: H160 = native_token;
price_estimator.estimate_prices(&tokens, denominator_token)
```
which means it always has price `1`. 
I think we should either document that in `collect_estimated_prices` and hard code the same `1` here or change it to always put the native token in the price hash map.

Reply:
Yes, I just noticed this. What do you prefer?

Reply:
(I'd go with adding it to the price vector since it avoids problems of changing it in one place and not the other later on, but I don't know if it has side effects)

Reply:
Agree. It is more robust to always have it in the price list. So I would say we document in `collect_estimated_prices` that the result will always contain the native token.